src/lib/CRSF/CRSF.cpp

```diff
@@ -370,9 +371,9 @@ void ICACHE_RAM_ATTR CRSF::GetChannelDataIn() // data is packed as 11 bits per c
     ChannelDataIn[13] = (rcChannels->ch13);
     ChannelDataIn[14] = (rcChannels->ch14);
     ChannelDataIn[15] = (rcChannels->ch15);
-        
+
     #if defined(PLATFORM_ESP32)
```

Comment:
Does PLATFORM_ESP8266 need to be added to this define and the other below for the devicesTriggerEvent() call?